### DISCUSSION TITLE
Is GPU meant to be blank in System Info when using DirectML?

### DISCUSSION THREAD

**Discussion by brknsoul** [vladmandic/sdnext#695] [Q&A] [answered]
The GPU line in the System Info tab is blank;
![image](https://user-images.githubusercontent.com/1969381/235609642-b0b3d7fd-297e-4bf4-8363-66d535db8b89.png)

It's definitely not running on CPU, since CPU gens take _ages_.

Is this normal behaviour when running in Windows 10?

Windows 10, AMD RX 6600 XT 8Gb.

**Comment by vladmandic** [comment, marked as answer]
DirectML uses DirectX to perform calculations. And DirectX can run on GPU or CPU.
But when using DirectML, there is no way to check internals - what is actually being used? To app, it "just works", it doesn't even know what its running on. 
If you have nVidia card with CUDA detected, app will try to use that. If you have AMD with ROCm, app will try to use that.
But if you have GPU or CPU that is not detected, running using DirectX may or may not be better than working purely on CPU - its basically "let DirectX decide".
